### PR TITLE
Add more documentation on configuration page and allow tags on all pages

### PR DESCRIPTION
**Pull request by diglin**

### CODE REVIEW

File: app/code/community/Mbiz/TrackingTags/etc/system.xml [hunk 1]
```diff
@@ -130,8 +158,50 @@
...
+Order ID: {{lastorder attr="id"}}
+Order increment ID: {{lastorder attr="increment_id"}}
+Order coupon : {{lastorder attr="coupon_code"}}
+Order items: {{lastorder items="id,sku,name,qty,qty_ordered"}}
+</pre>
+</code>
+                            ]]></comment>
                         </checkout_success_tag>
+                        <all_active translate="label" module="mbiz_trackingtags">
+                            <label>All Page tag</label>
```
Comment (jacquesbh) [inline app/code/community/Mbiz/TrackingTags/etc/system.xml]: `All pages tag` instead?

File: app/code/community/Mbiz/TrackingTags/etc/system.xml [hunk 2]
```diff
@@ -130,8 +158,50 @@
...
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <sort_order>900</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                            <comment><![CDATA[Active the tracking tag for all pages.]]></comment>
+                        </all_active>
+                        <all_tag translate="label" module="mbiz_trackingtags">
+                            <label>All Page tag</label>
```
Comment (jacquesbh) [inline app/code/community/Mbiz/TrackingTags/etc/system.xml]: `All pages tag` instead?

**Comment by diglin** [comment]
Updated

**Comment by jacquesbh** [comment]
Thanks!

**Pull request merged**